src/core/channels/encrypted_channel.ts

```diff
@@ -129,11 +129,13 @@ export default class EncryptedChannel extends PrivateChannel {
   }
 
   emitJSON(eventName: string, data?: any): Dispatcher {
```

Comment:
Would something like `emitFormattedData` make more sense? The product of this function doesn't really have to be JSON.

Reply:
That's a fair point.
I moved it around a little in e20f31b. I think that's clearer

Reply:
Beautiful! 🥇